libs/libcommon/include/common/iostream_debug_helpers.h

```diff
@@ -118,4 +130,19 @@ std::ostream & operator<<(std::ostream & stream, const std::chrono::time_point<c
 }
 
 
+#include <memory>
+
+template <class T>
+std::ostream & operator<<(std::ostream & stream, const std::shared_ptr<T> & what)
+{
+    stream << "std::shared_ptr{use_count=" << what.use_count() << ", ";
```

Comment:
Inconsistent use of round and curly braces. The logic must be obvious and clear or at least explained.